tests/lib/snaps/basic-iface-hooks-consumer/meta/hooks/configure

```diff
@@ -0,0 +1 @@
+#!/bin/sh
```

Comment:
(nitpick^2) Maybe a small comment here that the file is intentional empty and why? Like: `# empty, just needed to test the configure hook` or something.